packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporter.ts

```diff
@@ -0,0 +1,151 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as protoLoader from '@grpc/proto-loader';
+import * as grpc from 'grpc';
+import * as path from 'path';
+import * as collectorTypes from '../../types';
+
+import { ReadableSpan } from '@opentelemetry/tracing';
+import {
+  CollectorExporterBase,
+  CollectorExporterConfigBase,
+} from '../../CollectorExporterBase';
+import { CollectorExporterError } from '../../types';
+import { toCollectorExportTraceServiceRequest } from '../../transform';
+import { CollectorData, GRPCQueueItem } from './types';
+import { removeProtocol } from './util';
+
+const traceServiceClients: WeakMap<
```

Comment:
the main purpose of this was to keep a reference to collector, but because of refactor you don't need this anymore and could simply use `this` in code instead of 
`const exporter = traceServiceClients.get(this);`